content/it/docs/concepts/overview/components.md

```diff
@@ -1,5 +1,7 @@
 ---
 title: I componenti di Kubernetes
+description: >
+  Un cluster di Kubernetes è costituito da un insieme di componenti che sono, come minimo, un piano di controllo e uno o più sistemi di elaborazione, detti nodi.
```

Comment:
"Piano di controllo" which I believe would be the translation of "Control Plane" is not a term used in Italian in this meaning. Please, let's keep the English version (as is in the rest of the file)